src/panels/lovelace/cards/hui-logbook-card.ts

```diff
@@ -214,6 +178,55 @@ export class HuiLogbookCard extends LitElement implements LovelaceCard {
     `;
   }
 
+  private async _updateData(changedProperties: PropertyValues) {
+    if (!this._config || !this.hass) {
+      return;
+    }
+
+    if (this._fetchUserDone) {
+      await this._fetchUserDone;
```

Comment:
Why do we need to `await` this? We don't need it to do any of the stuff below here?

Reply:
@bramkragten The original idea was to copy the behavior of logbook panel, which [awaits for fetching users before render entries](https://github.com/home-assistant/frontend/blob/82a4806e01f9756a81a0117a1f7a05b126edef11/src/panels/logbook/ha-panel-logbook.ts#L265).
So that when the card shows entries, it's made sure that it will show user names.

But I realized that I did it not exactly right.
I made it wait for fetching users before start getting logbook entries.
I should have made it wait for fetching users AND getting logbook entries simultaneously (by using `Promise.all`).

**I fixed it, please take another look, thanks!**